README.md

```diff
@@ -37,6 +37,21 @@ driver and scripts to use AMD Vitis
 as a back-end for AMD FPGA using the open-source runtime and
 device-driver https://github.com/Xilinx/XRT
 
+
+## ArchGenMLIR
+
+The triSYCL project also contains part of the ArchGenMLIR
+tool. ArchGenMLIR is a tool to automatically generate approximations
+for mathematical fixed-point functions to optimize hardware usage for
+low precision computations. This was presented at the following conference:
+
+>  Luc FORGET, Gauthier HARNISCH, Ronan KERYELL and Florent DE
+>  DINECHIN. « A single-source C++ 20 HLS ﬂow for function evaluation
+>  on FPGA and beyond. » *In HEART2022: International Symposium on
+>  Highly-Efficient Accelerators and Reconﬁgurable Technologies*, pages
+>  51–58. Association for Computing Machinery, Tsukuba, Japan,
+>  June 2022. doi:10.1145/3535044. 3535051. https://hal.archives-ouvertes.fr/hal-03684757
+
```

Comment:
What we presented was ArchGen not ArchGenMLIR.
ArchGenMLIR is almost the same API but built with a better design on top of MLIR.